style.css

```diff
@@ -6,53 +6,13 @@ body {
 
 .container {
   display: flex;
-  justify-content: space-evenly;
-  flex-direction: row;
+  flex-direction: column;
+  align-items: center;
 }
 
-.column {
-  width: 33.3%;
-}
-
-/* #middle {
-  width: 33%;
-} */
-
-/*Centering Gif  from: https://www.sitepoint.com/css3-responsive-centered-image/ */
```

Comment:
There's a lot of outdated tips/tricks out there these days. I recommend sticking to stack overflow for finding similar problems, and verifying people's answers with MDN.